scripts/cv_api_master.py

```diff
@@ -101,6 +106,40 @@
     ],
 }
 
+BOUNDING_BOX_FORMATS = {
+        "path": "formats",
```

Comment:
Too much indentation

Reply:
Ran `black` on this file and it fixed it

Thanks!